.github/workflows/frontend-regression-tests.yml

```
@@ -0,0 +1,24 @@
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+        run:
+            working-directory: frontend
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)